### PR TITLE
Fix #3733

### LINKED ISSUE #3733: System freeze
While `sway-1.0-rc2` works perfectly, `sway-1.0-rc3` freeze the entire system on sway launch. My system has no logind support, so after compiling sway I have suid the sway binary.
### Configuration

- sway-1.0-rc3

- wlroots (Feb 20 2019, branch 'mastrer')

- Void GNU/Linux (Musl)

### PR DESCRIPTION
**Pull request by pixelherodev**

### CODE REVIEW

**Comment by RedSoxFan** [comment]
Can you please update the commit message to also have a standalone description of the issue and fix?

Otherwise, this LGTM

File: sway/config/output.c [hunk 1]
```diff
@@ -236,7 +236,7 @@ static bool spawn_swaybg(struct sway_output *output, char *const cmd[]) {
 			sway_log_errno(SWAY_ERROR, "execvp failed");
 			exit(EXIT_FAILURE);
```
Comment (emersion) [inline sway/config/output.c]: Can you replace all `exit` calls?

Comment (pixelherodev) [reply sway/config/output.c]: This good?

**Comment by pixelherodev** [comment]
> Can you please update the commit message to also have a standalone description of the issue and fix?
> 
> Otherwise, this LGTM

This good?

**Comment by emersion** [comment]
Thanks!

**Pull request merged**